t/chaos-test/go.mod

```diff
@@ -0,0 +1,5 @@
+module chaos
```

Comment:
Better to use a uri contains apisix, like
github.com/apache/apisix/t/chaos.

And `t/chaos/` might be a better name, as all files under `t` are tests.

Reply:
Thanks! Fixed